python/orion_finance_sdk_py/errors.py

```
@@ -0,0 +1,72 @@
+
+
+def _error_selector_map() -> dict[str, str]:
+    """Build 4-byte selector → error name from ErrorsLib (+ cached)."""
+    global _SELECTOR_TO_NAME
+    if _SELECTOR_TO_NAME is not None:
+        return _SELECTOR_TO_NAME
+
+    mapping: dict[str, str] = {}
+    try:
+        abi = load_contract_abi("ErrorsLib")
+    except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)